docs/forgedev/index.md

```diff
@@ -44,12 +44,12 @@ Depending on your favorite IDE, there is a different set of recommended steps yo
 
 Due to the way Eclipse workspaces work, ForgeGradle can do most of the work involved to get you started with a Forge workspace.
 
- 1. Open a terminal/command prompt and navigate to the directory of your cloned fork.
- 2. Type `./gradlew setup` and hit enter. Wait until ForgeGradle is done.
- 3. Type `./gradlew genEclipseRuns` and hit enter. Once again, wait until ForgeGradle is done.
- 4. Open your Eclipse workspace and go to `File -> Import -> General -> Existing Gradle Project`.
- 5. Browse to the repo directory for the "Project root directory" option in the dialog that opens.
- 7. Complete the import by clicking the "Finish" button.
+1. Open a terminal/command prompt and navigate to the directory of your cloned fork.
+2. Type `./gradlew setup` and hit enter. Wait until ForgeGradle is done.
+3. Type `./gradlew genEclipseRuns` and hit enter. Once again, wait until ForgeGradle is done.
+4. Open your Eclipse workspace and go to `File -> Import -> General -> Existing Gradle Project`.
+5. Browse to the repo directory for the "Project root directory" option in the dialog that opens.
+7. Complete the import by clicking the "Finish" button.
```

Comment:
The numbering of the last item in this list should be fixed to `6`. (Markdown renderers ignore the numbering in the source when rendering, which is why some choose to number ordered lists all using `1.`.)
```suggestion
6. Complete the import by clicking the "Finish" button.
```